src/pages/validator/external-chains/base.mdx

```diff
@@ -33,8 +33,22 @@ snap install docker
 git clone https://github.com/base-org/node
 ```
 
-2. In `docker-compose.yml`, set `OP_NODE_L1_ETH_RPC` to your own Goerli L1 RPC node. Using your own Goerli node is required to guarantee decentralization for validation.
+2. In `docker-compose.yml`, uncomment the right .env file according to the network you want to run it on
 
+```bash
+    env_file:
+      # select your network here:
+#      - .env.sepolia
+#      - .env.mainnet
+```
```

Comment:
This impacts Optimism as well (and potentially Mantle if you can take a look)

Reply:
Yes, will update other l2 too in a different PR